c-pallets/file-map/src/lib.rs

```diff
@@ -43,6 +43,9 @@ pub mod pallet {
 	#[scale_info(skip_type_params(T))]
 	#[codec(mel_bound())]
 	pub struct SchedulerInfo<T: pallet::Config> {
+		// Since IP or IPv6 have a fixed and know format, probably
+		// better to create a new IP struct that checks if it is a valid IP
+		// It is applicable to all `ip` references in the code
```

Comment:
I defined an enum for it:
https://github.com/CESSProject/cess/pull/88/files#diff-fba8d9a7246fa1a7499263ade4e960d909c56ddf5a3a423d7c763c423ef987b1R47
define here:
https://github.com/CESSProject/cess/pull/88/files#diff-1b48106f9cdd51de414e128717cc6b39dc2059cc0f9087f3c456ce63c2e4c6d1R37-R41